pkgs/desktops/gnome-3/devtools/gnome-builder/default.nix

```diff
@@ -0,0 +1,78 @@
+{stdenv
+, clang
+, devhelp
+, fetchgit
+, gettext
+, gspell
+, gnome_themes_standard
+, gtk3
+, gtksourceview
+, json_glib
+, jsonrpc_glib
+, libdazzle
+, libgit2-glib
+, libpeas
+, libxml2
+, llvmPackages
+, meson
+, ninja
+, python3
+, python3Packages
+, pcre
+, pkgconfig
+, pkgs
+, sysprof
+, template_glib
+, vala
+, vte
+, webkitgtk
+, wrapGAppsHook
+, ...}:
+
+let
+  version = "3.27.2";
+  name = "gnome-builder-${version}";
+  pythonPath = with python3Packages; python3Packages.makePythonPath [ libpeas pygobject3 ];
+in
+stdenv.mkDerivation
+{
+  name = name;
+  nativeBuildInputs = [ gettext meson ninja pkgconfig python3 wrapGAppsHook ];
+  buildInputs = [ devhelp gspell gtk3 gtksourceview json_glib jsonrpc_glib libdazzle libgit2-glib libpeas libxml2 llvmPackages.clang llvmPackages.llvm pcre sysprof template_glib vala vte webkitgtk ];
+
+  propagatedUserEnvPkgs = [ gnome_themes_standard ];
```

Comment:
I don't know if there is any way around it but try to avoid using `propagatgedUserEnvPkgs`.

Reply:
adding the theme as simple `buildInput` did solve it. Apparently there are some hooks that fixup things..